tests/baselines/reference/mappedTypeWithAsClauseAndLateBoundProperty.errors.txt

```diff
@@ -1,11 +1,11 @@

```

Comment:
This changes `[iterator]` to `[Symbol.iterator]` and `[unscopables]` to `[Symbol.unscopables]`. It seems like a good change to me.